### ISSUE TITLE
Development Environment

### ISSUE THREAD

**Issue by yoX64** [potomak/tomatoes#218]
I think it would be really useful to have a small tutorial of how to set up the application for development.

I saw that there is a link in the contributions file, but it does not work.

**Comment by potomak** [comment]
You're right. When I removed chef support in https://github.com/potomak/tomatoes/commit/850de0b12eb445e8e62bb6efb50179e36565ea41 I forgot to remove the link to the removed section from `CONTRIBUTING.md`.

**Comment by jwsloan** [comment]
Would you consider including a Docker file to achieve this goal? I had issues with my dev environment, and it would have been really handy to just use Docker to get up and running quickly.

**Comment by potomak** [comment]
@jwsloan definitely, there's already a task for that: https://github.com/tomatoes-app/tomatoes/issues/128

**Comment by madeindjs** [comment]
I added a small tutorial on #299. Take a look at [this file](https://github.com/tomatoes-app/tomatoes/pull/299/commits/1c37c16a4143016806107aa0ee0b65e0a60f98e0).

**Comment by yoX64** [comment]
Great!
